weblab_cg/__init__.py

```diff
@@ -29,7 +29,20 @@
     load_template,
 )
 
+from ._generator_chaste_cg import (   # noqa
+    NormalChasteModel,
+    OptChasteModel,
```

Comment:
Can we use more descriptive and more consistent names? What does `Opt` mean? And BEOpt? And why mix underscores with CamelCase?

Reply:
I want to keep to the same terminology as Chaste/PyCML so that we don't confuse everybody. I suppose the CamelCase comees from chast' c++-nes